chaos_genius/app.py

```diff
@@ -114,6 +118,24 @@ def shell_context():
     app.shell_context_processor(shell_context)
 
 
+def register_hooks(app):
+    @app.before_request
+    def authenticate():
+
+        if REACT_APP_IS_DEMO:
+            @app.before_request
+            def before_request():
+                if request.endpoint not in DEMO_ENDPOINT_WHITELIST:
+                    return jsonify({"status":"failure",
+                                    "message":"Endpoint not accessable in demo version"})
+                else:
+                    if request.method not in DEMO_ENDPOINT_WHITELIST[request.endpoint]:
+                        return jsonify({"status":"failure",
+                                        "message":"Method not allowed in demo version"})
```

Comment:
This message can be written better:
```suggestion
                                        "message": f"{request.method} method is not allowed for this endpoint in demo version"})
```